tests/conftest.py

```diff
@@ -191,6 +205,22 @@ async def trained_rasa_model(
     return trained_stack_model_path
 
 
+@pytest.fixture(scope="session")
+async def trained_simple_rasa_model(
```

Comment:
Does it bring some advantages over existing `trained_rasa_model()`?

Reply:
To be honest, `trained_rasa_model()` is not working on `e2e` branch right now. Something to do with entities. And I needed a way to test this.

Reply:
Currently we can not use `trained_rasa_model` here as there are a couple of different issues with how we fingerprint stories at the moment using `as_story_string()`.

These will be fixed in a separate PR.
So I think best for now to use this new `trained_simple_rasa_model`, and then unify it when we fix those bugs.